packages/components/text-link/src/TextLink.styles.ts

```diff
@@ -0,0 +1,112 @@
+import { css } from 'emotion';
+import type { CSSObject } from '@emotion/serialize';
```

Comment:
Looks like this dependency is not specified in package.json, so it can't really be used here